neqo-transport/src/recovery.rs

```diff
@@ -38,12 +54,13 @@ pub enum RecoveryToken {
 #[derive(Debug, Clone)]
 pub struct SentPacket {
     ack_eliciting: bool,
-    //in_flight: bool, // TODO needed only for cc
-    //size: u64, // TODO needed only for cc
     time_sent: Instant,
     pub(crate) tokens: Vec<RecoveryToken>,
 
     time_declared_lost: Option<Instant>,
+
+    in_flight: bool,
```

Comment:
I think that when we construct SentPacket, we should set this to true for every non-ack-eliciting packet.  Then for padded packets, we can have a `SentPacket::mark_in_flight(bool)` function we use to mark those as in-flight.  That's rare enough that we should fly past that most of the time.

We need the `mark_in_flight(false)` side for when something is removed from bytes_in_flight.

Reply:
I don't know why this would be needed, when we make the SentPacket we know if it has padding frames in it, and that it should therefore be marked in flight.